api/lib/domain/usecases/reconcile-schooling-registration.js

```diff
@@ -22,8 +23,34 @@ module.exports = async function reconcileSchoolingRegistration({
     userRepository,
     obfuscationService,
   });
+
   const student = await studentRepository.getReconciledStudentByNationalStudentId(matchedSchoolingRegistration.nationalStudentId);
   await userReconciliationService.checkIfStudentHasAlreadyAccountsReconciledInOtherOrganizations(student, userRepository, obfuscationService);
 
-  return schoolingRegistrationRepository.reconcileUserToSchoolingRegistration({ userId: reconciliationInfo.id, schoolingRegistrationId: matchedSchoolingRegistration.id });
+  await _checkIfAnotherStudentIsAlreadyReconciledWithTheSameOrganizationAndUser(reconciliationInfo.id, campaign.organizationId, schoolingRegistrationRepository);
+
+  return schoolingRegistrationRepository.reconcileUserToSchoolingRegistration({
+    userId: reconciliationInfo.id,
+    schoolingRegistrationId: matchedSchoolingRegistration.id,
+  });
+
 };
+
+async function _checkIfAnotherStudentIsAlreadyReconciledWithTheSameOrganizationAndUser(userId, organizationId, schoolingRegistrationRepository) {
```

Comment:
Je suis fatigué de me répéter: on réconcilie des `[utilisateurs](https://1024pix.atlassian.net/wiki/spaces/COM/pages/1022427205/Organisations+m+tier)` avec des `inscriptions`

```suggestion
async function _checkIfUserAlrreadyReconciledInOrganization(userId, organizationId, schoolingRegistrationRepository) {
```

Reply:
Vu ensemble au point tech, la proposition _checkIfUserAlreadyReconciledInOrganization peut illustrer deux cas de figures:

Le premier déjà géré dans le service de réconciliation:  checkIfStudentIsAlreadyReconciledOnTheSameOrganization.
Le deuxième qu'on a oublié: _checkIfAnotherStudentIsAlreadyReconciledWithTheSameOrganizationAndUser qui explicite le scénario difficilement reproductible.
Je prend cette proposition _checkIfAnotherStudentIsAlreadyReconciledInTheSameOrganization pour rester cohérent avec l'existant.